docs-markdown/src/test/suite/controllers/quick-pick-menu-controller.test.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 import * as chai from 'chai';
 import * as spies from 'chai-spies';
+import * as telemetry from '../../../helper/telemetry';
```

Comment:
It looks like this is throwing an error, as the telemetry import is already happening on line 23 of this file. Please remove this.

Reply:
It looks like someone beat me to the punch on this change,  another PR which was recently merged, see [here](https://github.com/microsoft/vscode-docs-authoring/commit/e372d608139a7f100390461a7c3d3d5f35d14405).  I've removed the duplicate import.  However the build check is still failing.  I've ran this locally on my Mac and on a Windows10 VM and the test pass.  Can you please share the details of the new failure?

Also there is a new check for CodeQL-Build which is failing with:


```
Run github/codeql-action/autobuild@v1
Attempting to automatically build csharp code
  /opt/hostedtoolcache/CodeQL/1.0.0/x64/codeql/csharp/tools/autobuild.sh
  Semmle autobuilder for C#
  Working directory: /home/runner/work/vscode-docs-authoring/vscode-docs-authoring
  Attempting to locate build script
  Error: Could not auto-detect a suitable build method
  ##[error]We were unable to automatically build your code. Please replace the call to the autobuild action with your custom build steps.  The process '/opt/hostedtoolcache/CodeQL/1.0.0/x64/codeql/csharp/tools/autobuild.sh' failed with exit code 1
```

I'm not familiar with this, can you please advise?

I'm not familiar with this, please advise.